src/easylink/devtools/implementation_creator.py

```diff
@@ -42,10 +46,11 @@ def __init__(self, script_path: Path, host: Path) -> None:
         self.script_path = script_path
         self.host = host
         self.recipe_path = script_path.with_suffix(".def")
-        self.container_path = script_path.with_suffix(".sif")
+        self.local_container_path = script_path.with_suffix(".sif")
+        self.hosted_container_path = self.host / self.local_container_path.name
         self.implementation_name = script_path.stem
         self.requirements = self._extract_requirements(script_path)
-        self.step_name = self._extract_step_name(script_path)
+        self.step = self._extract_step_being_implemented(script_path)
```

Comment:
Nit: rename to `_extract_implemented_step()`?

Reply:
words are hard